bin/remove_crossmapped_read_paires_BAM.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Typo in the file name - "paires -> pairs" I think ...?

Reply:
Good spotting! - done